WORKSPACE

```diff
@@ -15,7 +15,10 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-workspace(name = "graknlabs_protocol")
+workspace(
+    name = "graknlabs_protocol",
+    managed_directories = {"@npm": ["node_modules"]},
```

Comment:
As we've added node modules, we want to be able to use the modules as targets, which necessitates managing it as a directory.

Reply:
I feel like we should rename `@npm` to `@node_modules` for clarity - what do you think?

We would make the same change in `client-nodejs`.